src/parser/signup.js

```diff
@@ -1,10 +1,16 @@
 module.exports = function parseSignup(message, discordId) {
+  
+  if (typeof discordId === 'undefined') {
```

Comment:
I believe it's better to check if it's not a string; that's the type we want. I believe Discord IDs are also of fixed length; if they are, we should do a regex check to see:
- If they're the correct length
- If they only contain numbers.

This way we avoid getting extraneous things in the ID itself.